### PR TITLE
[miio] Create dreame.vacuum.p2036-miot.json

### PR DESCRIPTION
**Pull request by mabosch74**
added support for Trouver Finder Vacuum Robot

### CODE REVIEW

File: bundles/org.openhab.binding.miio/src/main/resources/database/dreame.vacuum.p2036-miot.json [hunk 1]
```diff
@@ -0,0 +1,1020 @@
+{
+	"deviceMapping": {
+		"id": [
+			"dreame.vacuum.p2036"
```
Comment (marcelrv) [inline bundles/org.openhab.binding.miio/src/main/resources/database/dreame.vacuum.p2036-miot.json:4]: I was faster :-) #10254 was just merged adding support for this device.

File: bundles/org.openhab.binding.miio/src/main/resources/database/dreame.vacuum.p2036-miot.json [hunk 2]
```diff
@@ -0,0 +1,1020 @@
...
+				"piid": 2,
+				"friendlyName": "Robot Cleaner - Device Fault",
+				"channel": "fault",
+				"type": "Number",
+				"stateDescription": {
+					"readOnly": true,
+					"options": [
+						{
+							"value": "0",
+							"label": "No Error"
```
Comment (marcelrv) [inline bundles/org.openhab.binding.miio/src/main/resources/database/dreame.vacuum.p2036-miot.json:155]: in #10254 the faults were not there. Can you rework these changes and include them  in dreame.vacuum.p2009-miot.json so they are available for all 4 robots. (these 4 robots have identical spec files, hence better to maintain 1 file than adding a 2nd one)

**Review by marcelrv — CHANGES_REQUESTED** [review]
Thanks for your contribution.
Please merge relevant changes from this file instead into dreame.vacuum.p2009-miot.json

File: bundles/org.openhab.binding.miio/src/main/resources/database/dreame.vacuum.p2036-miot.json (hunk 2 above)
Comment (mabosch74) [reply bundles/org.openhab.binding.miio/src/main/resources/database/dreame.vacuum.p2036-miot.json:155]: hey, marcel. in your post the starting with sweeping, set's siid=2 & aiid=2. this doesn't work on p2036. the vacuum starts only with siid=2 & aiid=1. I also include a "find me"-function on my post.

Comment (marcelrv) [reply bundles/org.openhab.binding.miio/src/main/resources/database/dreame.vacuum.p2036-miot.json:155]: Indeed, I see that is indeed a typo in the my file... even more reasons to merge your improvements in the p2009 file

Comment (mabosch74) [reply bundles/org.openhab.binding.miio/src/main/resources/database/dreame.vacuum.p2036-miot.json:155]: #10269 new pull request

**Pull request closed without merging**